src/utilities/gql.ts

```diff
@@ -1507,6 +1790,31 @@ const gql = {
     }
   `,
 
+  GET_PLAN_DERIVATION_GROUP: `#graphql
+    query GetPlanExternalSource($plan_id: Int!) {
+      links: ${Queries.PLAN_DERIVATION_GROUP}(where: {plan_id: {_eq: $plan_id}}) {
+        derivation_group_name
+        plan_id
+      }
+    }
+  `,
+
+  GET_PLAN_EVENT_TYPES: `#graphql
+    query GetPlanEventTypes($plan_id: Int!){
+      ${Queries.PLAN_DERIVATION_GROUP}(where: {plan_id: {_eq: $plan_id}}) {
+        ${Queries.DERIVATION_GROUP} {
```

Comment:
Is this actually a query? I would think this is just the return definition. Same with the next line.